.github/workflows/contributor.yaml

```diff
@@ -31,25 +47,25 @@ jobs:
           reactions: 'rocket'
 
   kubernetes-tests:
+    needs: trigger-contributor-tests
     name: "Contributor #${{ github.event.issue.number }}"
-    if: ${{ github.event.issue.pull_request && contains(github.event.comment.body, '/bot run tests') && contains(fromJson('["OWNER", "COLLABORATOR", "MEMBER"]'), github.event.comment.author_association)}}
-    uses: nebari-dev/nebari/.github/workflows/kubernetes_test.yaml@release/2022.11.1
+    uses: nebari-dev/nebari/.github/workflows/kubernetes_test.yaml@release/2023.4.1
```

Comment:
Driveby. We should probably add this to our release checklist.

Reply:
One second thought: we do we want to run the "released" tests at all. Shouldn't this be `@develop`? Otherwise no changes to the default branch will be considered when running contributor tests.

Reply:
@pmeier I agree, `@develop` is what we want.